PITCHME.md

```diff
@@ -29,14 +34,17 @@ An example of *WPS* file of Italy.
 
 An example of *INPUT* file.
 
+---?gist=380ea4af805494fc85dc668dff46ec12
+
+> NB Please, replace `<your@email.here>` with your own.
 
 ---
 
-# Demo
+## Demo
 
-![Video](https://www.youtube.com/embed/vSI9fEwghHw)
+![Video](https://www.youtube.com/embed/vFFi4Yei4Eo)
 
 ---
 
```

Comment:
What about having a section on "what to do if you get stuck"? Add a link to the forum to discuss the page. You can even embed the comments on the page from discourse...